packages/polaris-viz/src/components/DonutChart/components/LegendValues/LegendValues.tsx

```diff
@@ -0,0 +1,95 @@
+import type {ColorVisionInteractionMethods, DataSeries} from 'index';
+import type {LabelFormatter} from '@shopify/polaris-viz-core';
+import {useTheme} from '@shopify/polaris-viz-core';
+
+import {TrendIndicator} from '../../../TrendIndicator';
+import {getTrendIndicatorData} from '../../../../utilities/getTrendIndicatorData';
+import {SquareColorPreview} from '../../../../components/SquareColorPreview';
+
+import styles from './LegendValues.scss';
+
+interface LegendContentProps {
+  data: DataSeries[];
+  labelFormatter: LabelFormatter;
+  getColorVisionStyles: ColorVisionInteractionMethods['getColorVisionStyles'];
+  getColorVisionEventAttrs: ColorVisionInteractionMethods['getColorVisionEventAttrs'];
+}
+
+export function LegendValues({
+  data,
+  labelFormatter,
+  getColorVisionStyles,
+  getColorVisionEventAttrs,
+}: LegendContentProps) {
+  const selectedTheme = useTheme();
+
+  const maxTrendIndicatorWidth = data.reduce((maxWidth, {metadata}) => {
```

Comment:
Used this util `getTrendIndicatorData` to calculate the max trend indicator width so I could set the width of that column to the max trend with. Without this the trend indicators kept getting squished and the sizes were not consistent

<img width="268" alt="Screenshot 2023-12-06 at 10 20 01 AM" src="https://github.com/Shopify/polaris-viz/assets/90475364/1c67c31d-2d2b-4660-83e0-4ab178dfe852">

Reply:
@envex Sorry, that photo was meant to show what it looked like without setting the width on the table column

Reply:
ah do you mean the spacing between the values and trends?